src/CometWrapper.sol

```diff
@@ -115,9 +119,11 @@ contract CometWrapper is ERC4626, CometHelpers {
     function redeem(uint256 shares, address receiver, address owner) public override returns (uint256 assets) {
```

Comment:
nit: 

Part of me is wondering since we're pulling this project into our project base, I was wondering if we should change the coding style to match to our solidity coding style in Comet.sol 🤔 ?

In Comet.sol we do `return ABC;` statements instead of just assigning value to the variable that's declared in `returns(assets)`

Reply:
Good point, I'm not a fan of named return variables either. This is also a reason why the approval bug was possible. 

I'll make this change as part of a separate refactoring PR.

Reply:
Removed the named return variables in PR https://github.com/compound-finance/comet-wrapper/pull/4.